src/components/picker/picker.tsx

```diff
@@ -254,12 +267,17 @@ export class Picker {
                 style={{
                     width: `${boundingRect.width}px`,
                 }}
-                class="dropdown--list mdc-elevation-transition mdc-elevation--z4 mdc-menu-surface mdc-menu-surface--open"
+                class={`dropdown--list
+                        mdc-elevation-transition
+                        mdc-elevation--z4
+                        mdc-menu-surface mdc-menu-surface--open
+                        ${this.displayFullList ? 'display-full-list' : ''}	
+                        `}
```

Comment:
Python-style 😄

I think we've mostly been using the style below:

```tsx
class={`
    dropdown--list
    mdc-elevation-transition
    mdc-elevation--z4
    mdc-menu-surface mdc-menu-surface--open
    ${this.displayFullList ? 'display-full-list' : ''}
`}
```

It makes no difference whatsoever, so it's just a matter of aesthetic preference. I'll push a fixup of it for consistency though.

Reply:
⚡️